yarn.lock

```diff
@@ -3994,6 +3994,18 @@ vite@2.5.0:
   optionalDependencies:
     fsevents "~2.3.2"
 
+vite@2.7.13:
```

Comment:
@addisonschultz I think we sometimes merge PRs without running `yarn` on them which makes `yarn.lock` file out of sync & that's why these changes appear randomly in other PRs.